src/videojs.ima.js

```diff
@@ -428,6 +428,7 @@
         fullscreenDiv.className = 'ima-fullscreen';
         adContainerDiv.style.width = window.screen.width + 'px';
         adContainerDiv.style.height = window.screen.height + 'px';
+        adContainerDiv.className = 'ima-ad-container--fullscreen';
```

Comment:
actually, no longer have access to the guardian's repo. maybe @sndrs wouldn't mind changing it...